ejviola/mailroom.py

```diff
@@ -49,7 +58,15 @@ def send_thank_you():
             input("Are you thanking " + str(user_thank_you_input) + " for a new donation, or sending a followup thank-you for an old donation? Enter new or old:\n"))
         if thank_you_path == str("new"):
             # If it is a new donation, send a thank you based on the details of that donation
-            amount_donated = int(input("Please enter the amount " + user_thank_you_input + " donated (e.g. '10000'):\n"))
+            # While loop to make sure the amount donated is valid
+            while True:
+                try:
+                    amount_donated = input("Please enter the amount " + user_thank_you_input + " donated (e.g. '10000'):\n")
+                    amount_donated = float(amount_donated)
+                    break
+                except ValueError:
+                    print("You did not enter a valid amount.")
+                    pass
             donation_date = str(input("Please enter the date the donation was made (e.g. 'January 1, 2017': \n"))
             donation_cause = str(input(
                 "Please enter the cause that the donation will support (e.g. 'increase rural broadband access'\n"))
```

Comment:
Anything from the `input` function will result in a string, so no need to cast the result with `str`